lib/paperclip/storage/s3.rb

```diff
@@ -446,10 +446,10 @@ def flush_deletes #:nodoc:
         @queued_for_delete = []
       end
 
-      def copy_to_local_file(style, local_dest_path)
+      def copy_to_local_file(local_dest_path, style_name = default_style)
```

Comment:
We don't need to switch the order of parameters, can we leave them as they were? Thank you.